hypershift-operator/controllers/nodepool/config.go

```diff
@@ -319,3 +321,35 @@ func encode(obj runtime.Object, ser *serializer.Serializer) ([]byte, error) {
 	}
 	return buff.Bytes(), nil
 }
+
+func globalConfigString(hcluster *hyperv1.HostedCluster) (string, error) {
```

Comment:
why are we moving this function? seems like more diff that needed.

Reply:
It's actually left over of previous refactor. The func belongs to config.go where is being consumed. I put a dedicated commit for this now.